ext/pgsql/pgsql.c

```diff
@@ -6324,7 +6325,7 @@ PHP_FUNCTION(pg_close_stmt)
 
 	ZEND_PARSE_PARAMETERS_START(2, 2)
 		Z_PARAM_OBJECT_OF_CLASS(pgsql_link, pgsql_link_ce)
-		Z_PARAM_STR(stmt)
+		Z_PARAM_PATH_STR(stmt)
 	ZEND_PARSE_PARAMETERS_END();
 
 	if (ZSTR_LEN(stmt) == 0) {
```

Comment:
Nit I just saw, could you update the below argument value error to use the helper? Can be a follow-up PR.

Reply:
no pb I can do it now it s nothing.